components/brave_shields/browser/ad_block_engine.cc

```diff
@@ -273,6 +278,17 @@ void AdBlockEngine::AddKnownTagsToAdBlockInstance() {
   });
 }
 
+void AdBlockEngine::OnFilterSetLoaded(rust::Box<adblock::FilterSet> filter_set,
+                                      const std::string& resources_json) {
+  auto result = adblock::engine_from_filter_set(std::move(filter_set));
+  if (result.result_kind != adblock::ResultKind::Success) {
+    LOG(ERROR) << "AdBlockEngine::OnFilterSetLoaded failed: "
```

Comment:
Same as the other one, logging like this generally isn't useful so we would normally restrict this to a VLOG. We shouldn't just be logging errors into the main chrome log and actually I think even that is disabled by default maybe